### PR TITLE
Add stub test-e2e-encryption suite

### PR DESCRIPTION
**Pull request by enj**
Signed-off-by: Monis Khan <mkhan@redhat.com>

So that I can have real long running disruptive e2e tests for #516

### CODE REVIEW

**Comment by enj** [comment]
@mfojtik @sttts @p0lyn0mial

File: Makefile [hunk 1]
```diff
@@ -42,6 +42,15 @@ test-e2e: GO_TEST_PACKAGES :=./test/e2e/...
...
 test-e2e: test-unit
 
+# these are extremely slow serial e2e encryption tests that modify the cluster's global state
+.PHONY: test-e2e-encryption
+test-e2e-encryption: GO_TEST_PACKAGES :=./test/e2e-encryption/...
+test-e2e-encryption: GO_TEST_FLAGS += -v
+test-e2e-encryption: GO_TEST_FLAGS += -timeout 3h
+test-e2e-encryption: GO_TEST_FLAGS += -p 1
+test-e2e-encryption: GO_TEST_FLAGS += -parallel 1
+test-e2e-encryption: test-unit
```
Comment (sttts) [inline Makefile]: how is this picked up? do we need an openshift/release job?

Comment (enj) [reply Makefile]: https://github.com/openshift/release/pull/5026

**Pull request merged**